main.js

```diff
@@ -0,0 +1,101 @@
+let beginBoard;
+let startButton = document.getElementById('start')
+let status = document.getElementById('status')
+
+startButton.addEventListener('click', function() {
+    event.target.disabled = true;
+    event.target.textContent = "Player X's turn"
+})
+
+    const humanPlayer = 'X';
+    const computerPlayer = 'O';
+
+    const winConditions = [
+        [0, 1, 2], // row
+        [3, 4, 5], // row
+        [6, 7, 8], // row
+        [0, 3, 6], // column
+        [1, 4, 7], // column
+        [2, 5, 8], // column
+        [0, 4, 8], // diagonal
+        [6, 4, 2]  // diagonal
+    ]
+
+const cells = document.querySelectorAll ('.cell'); // .cell or .board?
+startGame();
+
+function startGame() {
+    document.querySelector (".endgame").getElementsByClassName.display = "none";
+        Array.from(Array(9).keys());
+        for (let i = 0; i < cells.length; i++) {
+            cells[i].innerText = '';
+            cells[i].style.removeProperty ('background-color');
+            cells[i].addEventListener ('click', onTurn, false);
+        }
+}
+
+function onTurn(square) {
+    if (typeof beginBoard [square.target.id] === 'number') {
+        turn (square.target.id, humanPlayer)
+        if (!isTied()) turn (goodSpot(), computerPlayer);
+    }
+}
+
+function turn (squareId, player) {
+    beginBoard [squareId] = player;
+    document.getElementById(squareId).innerText = player;
+    let itsWon = isWin (beginBoard, player)
+        if (itsWon) itsOver (itsWon)
+}
```

Comment:
You can determine which cell is being clicked a little easier by accessing the `event.target` inside your event handler callback, which is the element the event was triggered on